src/containers/PhotoBoard.jsx

```diff
@@ -113,4 +115,4 @@ const mapDispatchToProps = (dispatch, ownProps) => ({
 export default connect(
   mapStateToProps,
   mapDispatchToProps
-)(translate()(PhotoBoard))
+)(translate()(Dimensions()(PhotoBoard)))
```

Comment:
Regarding previous discussion, it seems legit to `Dimensions()` the `PhotoBoard`, what do you think @gregorylegarec ?